spec/game_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'spec_helper'
+require_relative '../dependencies'
+
+RSpec.describe Game do
+  subject(:game) { described_class.new(name: 'Rspec', difficulty: :easy) }
+
+  describe '.check_numbers' do
```

Comment:
do not test private methods
test this logic in by calling public method `check`